src/components/ui/Chat/MessageList/styleMessageList.tsx

```diff
@@ -0,0 +1,111 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Nit on the name: usually we identify a styled component with a 'Styled' prefix. It feels a bit confusing to have this name that is similar.

Reply:
Also for some reason there's a lack of semicolons in this file, I won't point out each instance.

Reply:
Yeah, I will update the name and add in the semicolons.